sensorlib/src/objects/DetectedObject.py

```diff
@@ -35,12 +35,23 @@ class DetectedObject:
 class DetectedObjectBuilder:
     @staticmethod
     def build_detected_object(carla_actor, allowed_semantic_tags):
+        object_type = CarlaUtils.determine_object_type(carla_actor, allowed_semantic_tags)
+        #print("In builder: detected object type: " + object_type)
```

Comment:
Using a logger instead of print statement would avoid commenting in and out print statements all over this code.

Reply:
removed all the print logs